core/src/main/scala/io/scalac/mesmer/core/util/Timestamp.scala

```diff
@@ -10,6 +10,10 @@ final class Interval(private val nanos: Long) extends AnyVal {
   def toNano: Long   = nanos
 }
 
+object Interval {
```

Comment:
This is similar to what we talked about before, in #418 right? There's no need for Interval.toMillis and we can use new Interval(x).toMillis()